wireguard-manager.sh

```diff
@@ -43,8 +43,8 @@ function installing-system-requirements() {
         fi
         yum install curl coreutils jq iproute lsof cronie gawk procps-ng grep qrencode sed zip unzip openssl nftables NetworkManager e2fsprogs gnupg systemd -y
       elif { [ "${CURRENT_DISTRO}" == "arch" ] || [ "${CURRENT_DISTRO}" == "archarm" ] || [ "${CURRENT_DISTRO}" == "manjaro" ]; }; then
-        pacman -Sy
-        pacman -S --noconfirm --needed curl coreutils jq iproute2 lsof cronie gawk procps-ng grep qrencode sed zip unzip openssl nftables ifupdown e2fsprogs gnupg systemd
+        pacman -Sy archlinux-keyring --noconfirm
```

Comment:
It might be generally advisable to not run this on every start to prevent constant updates.

Reply:
Thanks, that cleared it up :) I'll submit a v2